multicluster/cmd/service-mirror/main.go

```diff
@@ -164,17 +165,26 @@ func Main(args []string) {
 						}
 					}
 				},
-				UpdateFunc: func(_, obj interface{}) {
-					link, ok := obj.(*v1alpha2.Link)
+				UpdateFunc: func(oldObj, currentObj interface{}) {
+					oldLink, ok := oldObj.(*v1alpha2.Link)
 					if !ok {
-						log.Errorf("object is not a Link: %+v", obj)
+						log.Errorf("object is not a Link: %+v", oldObj)
 						return
 					}
-					if link.GetName() == linkName {
+					currentLink, ok := currentObj.(*v1alpha2.Link)
+					if !ok {
+						log.Errorf("object is not a Link: %+v", currentObj)
+						return
+					}
+					if reflect.DeepEqual(oldLink.Spec, currentLink.Spec) {
+						log.Infof("Link update ignored (only status changed): %s", currentLink.GetName())
```

Comment:
Is it appropriate to log this at info? What information does it convey to users vs how frequently might this be emitted?

My sense is that this may log regularly/frequently and pollute logs.

```suggestion
						log.Debugf("Link update ignored (only status changed): %s", currentLink.GetName())
```